<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectContainer</class>
 <widget class="QDialog" name="selectContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目选择框</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#selectContainer
{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1,1,0,10,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="l_title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18px &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>项目选择框</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">border-top: 1px solid rgb(238, 238, 238); </string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="l_serch">
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>字符匹配：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_filterPattern">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
	border:1px solid rgb(149, 191, 255);
	background-color: rgb(255, 255, 255);
	border-top-left-radius:2px;
	border-bottom-left-radius:2px;
	border-top-right-radius:0px;
	border-bottom-right-radius:0px;
}

QToolButton::menu-indicator 
{
    image: none;
}</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/image/image/downArrow_toolButton.png</normaloff>:/image/image/downArrow_toolButton.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_filterText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border:1px solid rgb(149, 191, 255);
background-color: rgb(255, 255, 255);
border-top-right-radius:2px;
border-bottom-right-radius:2px;
border-top-left-radius:0px;
border-bottom-left-radius:0px;</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_showType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	font: 12px &quot;黑体&quot;;
	outline:0px;
}
QRadioButton::indicator
{
	width: 22px;
    height: 22px;
}

QRadioButton::indicator:unchecked
{
	image:url(:/image/image/switch_off.png);
}

QRadioButton::indicator:checked
{
	image:url(:/image/image/switch_on.png);
}
</string>
        </property>
        <property name="text">
         <string>显示未下载项目</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_filter" native="true">
     <property name="styleSheet">
      <string notr="true">QDateEdit{
	border:1px solid rgb(149, 191, 255);
	border-radius:2px;
	background-color: rgb(255, 255, 255);
}

QDateEdit::down-arrow
{
	image:none;
}

QDateEdit::drop-down {
	width: 20px;
	image:url(:/image/image/calender.png);
	background-color: rgb(255, 255, 255);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="l_createUser">
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>创建者：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbb_createUser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox
{
	border:1px solid rgb(149, 191, 255);
	border-radius:2px;
	background-color: rgb(255, 255, 255);
}

QComboBox:hover{
	border:1px solid rgb(93, 193, 252);
}

QComboBox::drop-down 
{
	border:none;
}

QComboBox::down-arrow 
{
    width: 9px;  
    height: 9px;   
    image: url(:/image/image/downArrow_toolButton.png);
}

QComboBox QAbstractItemView 
{
    outline: 0px solid gray;  /*取消选中虚线*/
    border: 1px solid rgb(31,156,220);  
    background-color: white;   
    selection-background-color: #1E90FF;   
}</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_span_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_from">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>起始时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_fromDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_span">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_to">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12px &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_toDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2099</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="styleSheet">
      <string notr="true">border-top: 1px solid rgb(238, 238, 238); </string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="tv_prj">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section { 
	height:25px;
	color:#505050;
	background:white;
	border:none;
}
QTreeView {
	background-color: rgb(255, 255, 255);
 	alternate-background-color: rgb(243, 244, 246);
	border:1px solid #eeeeee;
	show-decoration-selected:1;
}
QTreeView::item{
	border:none;
}
QTreeView::item:hover {
    background: rgb(234, 242, 255);
}
QTreeView::item:selected{
	background: #1E90FF;
}</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 12px &quot;黑体&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(53, 150, 251);
	border:none;
	border-radius:4px;
}
QPushButton:hover{
	background-color: rgb(75, 175, 255);
}
</string>
        </property>
        <property name="text">
         <string>选定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 12px &quot;黑体&quot;;
	background-color: rgb(255, 255, 255);
	border:1px solid rgb(149, 191, 255);
	border-radius:4px;
}
QPushButton:hover{
	border:1px solid rgb(93, 193, 252);
}
</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
